README.md

```diff
@@ -27,6 +27,23 @@ Please refer to the [developer guide](https://www.osbuild.org/docs/developer-gui
 
 See also the [local developer documentation](./docs/developer) for useful information about working with this specific project.
 
+#### YAML based image definitions
+
+More and more parts of the library are converted to use yaml to define core
+parts of an image. See this [example](./pkg/distro/packagesets/fedora/)
+directory. For local development that just changes the YAML based
+definitions the library can be forced to use alternative yaml dirs.
+
+E.g. if there is a `./my-yaml/fedora/package-sets.yaml` then that can be
+used via:
+```console
+$ IMAGE_BUILDER_EXPERIMENTAL=yamldir=./my-yaml image-builder build minimal-raw --distro fedora-42
+```
+Note that this will eventually become a switch for the `image-builder` tool
+but for now this environment option is required.
+
```

Comment:
I would expand this to include a very clear note that this will change in the future and is totally experimental and unsupported.

Reply:
I reworded this, please let me know what you think and if it should still be stronger.